internal/k8s/secret.go

```diff
@@ -23,11 +23,17 @@ const (
 
 // ValidateTLSSecret validates the secret. If it is valid, the function returns nil.
 func ValidateTLSSecret(secret *v1.Secret) error {
-	if _, exists := secret.Data[v1.TLSCertKey]; !exists {
+	_, certExists := secret.Data[v1.TLSCertKey]
+	_, keyExists := secret.Data[v1.TLSPrivateKeyKey]
+
+	if certExists && keyExists {
```

Comment:
because of this check, I think line 40 (`return nil`) is never invoked

Reply:
yeah but it's complaining if I don't add a return at the end 😅